pkg/lint/runner.go

```diff
@@ -72,6 +72,7 @@ func NewRunner(cfg *config.Config, log logutils.Log, goenv *goutil.Env, es *lint
 			processors.NewNolint(log.Child("nolint"), dbManager, enabledLinters),
 
 			processors.NewUniqByLine(cfg),
+			processors.NewSortResults(cfg),
```

Comment:
any reason sorting processor is added in this order? I was thinking about having it last

Reply:
That makes sense!